instrumentation/sequel/test/opentelemetry/instrumentation/sequel_test.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+describe OpenTelemetry::Instrumentation::Sequel do
+  let(:instrumentation) { OpenTelemetry::Instrumentation::Sequel::Instrumentation.instance }
+  let(:exporter) { EXPORTER }
+  let(:database) { Sequel.connect('sqlite::memory:') }
+  let(:model) { Class.new(Sequel::Model(:documents)) }
+
+  # class CustomError < StandardError; end
+
+  before do
+    instrumentation.install
+    database.extension :opentelemetry
+    database.create_table!(:documents) do
+      primary_key :id
+      String :title
+    end
+    model # Instantiate the model so that it doesn't interfere with further calls
+
+    exporter.reset
+  end
+
+  describe 'tracing' do
```

Comment:
The tests do not seem very specific to me or help me understand as an end user what I should expect to see on a span when I use this. 

The tests for `traces dataset executes` e.g. may continue to pass even though the span does not have the data we expect. 

May I ask that you expand on these tests (although it may seem verbose) to include assertions around the spans that were captured?

E.g. name, kind, attributes, if any events where captured etc..

Reply:
I've made some changes, but will expand it even further.